mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerator
```

Comment:
I believe this is the abstracted interface/abstract class.

Would be nice to have some quick javadoc on the class itself.

Reply:
```suggestion
CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerator
	COMMENT An interface describing random behavior in Minecraft.
	COMMENT
	COMMENT This interface should be used instead of {@link java.util.Random} for random number generation.
```

Reply:
third line of the diff should be
`	COMMENT <p>`

Reply:
look good now?

Reply:
yup, I guess